### PR TITLE
updated RDMS feature set

### PR DESCRIPTION
**Pull request by smendis-scottlogic**

### CODE REVIEW

**Review by mlysaght2017 — DISMISSED** [review]

File: services/common-features.yaml [hunk 1]
```diff
@@ -118,3 +117,13 @@ features:
     description: |
       Provides the ability to copy data or resource to multiple locations to ensure
       availability and durability.
+
+  - id: CCC.F22 # Location Lock-In
```
Comment (sshiells-scottlogic) [inline services/common-features.yaml:121]: Can you add these to the common features snippets?

Comment (smendis-scottlogic) [reply services/common-features.yaml:121]: Added

**Review by sshiells-scottlogic — APPROVED** [review]

**Comment by smendis-scottlogic** [comment]
@damienjburks Will you be able to take a look at this PR please?

**Review by damienjburks — APPROVED** [review]
LGTM 🚀

**Review by sshiells-scottlogic — APPROVED** [review]

**Pull request merged**